region_migrations/refresher.go

```diff
@@ -93,16 +96,28 @@ func (r *Refresher) migrationRefresher() error {
 	client := r.client
 	r.lock.Unlock()
 
+	errCount := 0
+
+	var migration *scalingo.RegionMigration
+
 	for {
```

Comment:
Could you use `go-utils/retry` ?

Reply:
I thought about it and it wont be easy here. There's no mechanisme to know where we are in the retry. And the code still need to run. This will need a major redesign of this part and to add very different functionnalities to this package. Not sure this is worth the time.

Reply:
It would be at least interesting to list missing features in the package.

Like a callback with the iteration number and some other information or something like that, don't you think?

Reply:
(I'd be glad to have a quite polyvalent retry package)

Reply:
Ok i'll see